app/ide-desktop/lib/client/src/ipc.ts

```diff
@@ -12,4 +14,10 @@ export const channel = {
     profilesLoaded: 'profiles-loaded',
     saveProfile: 'save-profile',
     quit: 'quit-ide',
+    /** Channel for requesting that a URL be opened by the system browser. */
+    openUrlInSystemBrowser: 'open-url-in-system-browser',
+    /** Channel for setting a callback that handles deep links to this application. */
+    setDeepLinkHandler: 'set-deep-link-handler',
```

Comment:
I don't see this variable used anywhere in the code.

Reply:
This will be used by the subsequent PR. The dashboard is responsible for setting a handler, since it's the module that knows where/how to route the authentication pages.